pennylane/fermi/fermionic.py

```diff
@@ -108,8 +108,67 @@ def __repr__(self):
         r"""Terminal representation of a FermiWord"""
         return str(self)
 
+    def __add__(self, other):
+        """Add a FermiSentence, FermiWord or constant to a FermiWord. Converts both
+        elements to be added into FermiSentences, and uses the FermiSentence __add__
+        method"""
+
+        self_fs = FermiSentence({self: 1.0})
+
+        if isinstance(other, FermiSentence):
+            return self_fs + other
+
+        if isinstance(other, FermiWord):
+            other_fs = FermiSentence({other: 1.0})
+            return self_fs + other_fs
+
+        if isinstance(other, (float, int, complex)):
+            other_fs = FermiSentence({FermiWord({}): other})
+            return self_fs + other_fs
+
+        raise TypeError(f"Cannot add {type(other)} to a FermiWord.")
+
+    def __radd__(self, other):
+        """Add a FermiWord to a constant, i.e. `2 + FermiWord({...})`"""
```

Comment:
How about adding a `FermiSentence` to a `FermiWord`? That is channelled by the `FermiSentence` `__add__ `method?

Reply:
yes :)